test/src/pages/details/get-this-title/recommended-callout.test.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import {render, screen} from '@testing-library/preact';
+import {LanguageContextProvider} from '~/contexts/language';
+import RecommendedCallout from '~/pages/details/common/get-this-title-files/recommended-callout/recommended-callout';
+import BookDetailsLoader from '../book-details-context';
+
+const onPutAway = jest.fn();
+
+function LangWrapRecommendedCallout({...args}) {
+    return (
+        <LanguageContextProvider>
+            <BookDetailsLoader slug='books/college-algebra'>
+                <RecommendedCallout onPutAway={onPutAway} {...args} />
+            </BookDetailsLoader>
+        </LanguageContextProvider>
+    );
+}
+
+describe('details/recommended-callout', () => {
```

Comment:
Wrapped the tests in describe, which made it a major change (due to indenting everything)